app/scripts/HorizontalMultivecTrack.js

```diff
@@ -78,8 +78,7 @@ export default class HorizontalMultivecTrack extends HeatmapTiledPixiTrack {
 
     this.zoomLevel = this.calculateZoomLevel();
 
-
-    if (this.tilesetInfo.resolution) {
+    if (this.tilesetInfo.resolutions) {
```

Comment:
Is this an accidental change or related to a bug?

Reply:
Shit, I have no idea. I completely forgot what these changes were for. I must have stacked this PR on top of another one.

Reply:
Oh, so this was just a typo: `this.tilesetInfo.resolution` should be `this.tilesetInfo.resolutions`.